Library/Homebrew/dev-cmd/linkage.rb

```diff
@@ -84,6 +88,21 @@ def display_normal_output
       display_items "Possible undeclared dependencies", @undeclared_deps
     end
 
+    def display_reverse_output
+      return if @reverse_links.empty?
+      puts "Links to:"
+      sorted = @reverse_links.sort
+      sorted.each do |dylib, files|
+        next if dylib == "/usr/lib/libSystem.B.dylib"
+        puts dylib
+        files.each do |f|
+          unprefixed = f.sub "#{@keg.to_s}/", ""
```

Comment:
Use `f.strip_prefix("#{@keg}/")`. (Notice also there's no need for the `.to_s` when using a variable with string interpolation.)